backend/shigai-serve/src/main.rs

```diff
@@ -0,0 +1,127 @@
+use std::io::prelude::{Read, Write};
+use std::net::TcpListener;
+use std::net::TcpStream;
+
+const SRC_ROOT: &str = "../../frontend/src";
```

Comment:
This is good for testing, but we'll want some kind of build process for the real thing in the long run

Reply:
save for a future feature